website/docs/users-sources/sources/social-logins/azure-ad/index.md

```diff
@@ -111,3 +111,22 @@ return True
 9. Open **Flow settings** and choose _azure-ad-enrollment_ as enrollment flow.
 
 Try to login with a **_new_** user. You should see no prompts and the user should have the correct information.
+
+### Machine-to-machine authentication <span class="badge badge--version">authentik 2024.12+</span>
+
+When doing [Machine-to-Machine](../../../../add-secure-apps/providers/oauth2/client_credentials.md#jwt-authentication), some specific steps need to be considered.
+
+When getting the JWT token from Azure AD, the scope needs to be set to the Application ID URI, and _not_ the Graph URL, as otherwise the JWT will be in an invalid format.
```

Comment:
```suggestion
When getting the JWT token from Azure AD, set the scope to the Application ID URI, and _not_ the Graph URL; otherwise the JWT will be in an invalid format.
```